### PR TITLE
changed freeze progress bar to progressBarStyleHorizontal

### PR DESCRIPTION
**Pull request by michaone007**

### CODE REVIEW

**Comment by michaone007** [comment]
ready to merge

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 1]
```diff
@@ -402,11 +399,29 @@ public void displayCandies() {
     }
 
     public void freezeProgressBar() {
+        freezeBar.getProgressDrawable().setColorFilter(Color.parseColor("#9254ff"),android.graphics.PorterDuff.Mode.SRC_IN);
         freezeBar.setVisibility(View.VISIBLE);
-        ObjectAnimator anim = ObjectAnimator.ofInt(freezeBar, "progress", 0, 100);
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java:412]: can't we use this approach in the horizontal line?

Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:412]: do you mean the color?

Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:412]: cuz i didnt see any xml solution to that

Comment (zakiva) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:412]: not the color, all the `ObjectAnimator` thing

Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:412]: i erased them................ its in red color

Comment (zakiva) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:412]: I know! that's exactly what I'm asking here: 
"can't we use this approach - aka using the `ObjectAnimator` - in the horizontal line?"

Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:412]: ohhhh i will see ..

**Comment by michaone007** [comment]
ready to merge

**Comment by zakiva** [comment]
@michaone007 need rebase

**Comment by michaone007** [comment]
Ready to merge

**Pull request merged**